.github/workflows/build-images.yml

```diff
@@ -312,10 +311,15 @@ jobs:
       uses: ./.github/actions/download-rpm
       if: ${{ inputs.v2_vertica_image == '' }}
 
-    - name: Build and optionally push v2 server image
+    - name: Build the v2 image
       if: ${{ inputs.v2_vertica_image == '' }}
       run: |
-        export VERTICA_IMG=${{ steps.v2_vertica_image.outputs.value }}
+        # We never push the image we build here because the RPM we use is too
+        # old. The RPM needs to be from at least a 23.3.0 server. So, we build
+        # here only to test the creation of the image. We don't verify this
+        # image in the remaining CI. Pick a dummy image name so as not to
+        # overwrite the output build image that the e2e tests will use.
+        export VERTICA_IMG=vertica-k8s:kind-v2
         make docker-build-vertica-v2
         if [ $GITHUB_EVENT_NAME != 'pull_request' ]
```

Comment:
We should remove this push because we aren't going to use this new image anywhere in the workflow.